horizons/gui/widgets/minimap.py

```diff
@@ -34,70 +34,44 @@
 from horizons.util.shapes import Circle, Point, Rect
 
 
-def iter_minimap_points(location, world, island_color, water_color, area=None):
+def get_minimap_color(world_coords, world, island_color, water_color):
```

Comment:
Could you please add a docstring documenting what the function does specifically?
`iter_minimap_points_colors` is nicely documented, but `get_minimap_color `isn't.